src/commandHandlers/fallback.ts

```diff
@@ -1,11 +1,28 @@
 import { MessageEmbed } from 'discord.js';
 
 import config from '../config';
+import { suggestSimilarCommands } from '../suggestSimilarCommands';
 
 const { COMMAND_PREFIX } = config;
 
 export const fallback = async (arg: string, embed: MessageEmbed) => {
-  return embed
-    .setTitle('ERROR: ooops...command not found')
-    .setDescription(`Try using the ${COMMAND_PREFIX}help command.`);
+  const command = arg;
+  const similarCommands = suggestSimilarCommands(command);
+
+  embed
+      .setTitle('ERROR: ooops...command not found');
+
+  if (similarCommands && similarCommands.length) {
+      embed
+          .setDescription('Here is a list of similar commands');
+
+      similarCommands
+          .forEach((commandItem: {command: (arg: string, embed: MessageEmbed) => Promise<MessageEmbed>, description: string, triggers: string[], usage: string}) => embed
```

Comment:
what do you think of adding a type for a `commandItem` instead of it just being in this function? Like
```ts
export interface CommandHandler {
    command: (arg: string, embed: MessageEmbed, message: Message) => Promise<MessageEmbed>
    description: string
    triggers: string[]
    usage: string
}
```

You can see more details on the interface for the command on the [docs](https://github.com/EddieJaoudeCommunity/EddieBot#how-to-add-a-new-command-to-the-bot) 😃

Reply:
@BOLT04 that would be absolutely awesome! I'll try to incorporate that. :smiley: